libs/wlgen/wlgen/rta.py

```diff
@@ -266,19 +266,29 @@ def _confCustom(self):
 
         self.json = '{0:s}_{1:02d}.json'.format(self.name, self.exc_id)
         ofile = open(self.json, 'w')
-        ifile = open(self.params['custom'], 'r')
         replacements = {
             '__DURATION__' : str(self.duration),
             '__PVALUE__'   : str(calibration),
             '__LOGDIR__'   : str(self.run_dir),
             '__WORKDIR__'  : '"'+self.target.working_directory+'"',
         }
 
+	# check for inline config
+	if 'custom_config' in self.params['custom']:
+        # inline config present, turn it into a file repr so we can parse the same
+        tmp_json = json.dumps(self.params['custom']['custom_config'],
+            indent=4, separators=(',', ': '), sort_keys=True)
+        ifile = tmp_json.split('\n')
+	else:
+        # open the supplied filename
+        ifile = open(self.params['custom']['custom_filename'], 'r')
+
```

Comment:
that's odd, there are 8 spaces before both those lines. what settings are you after?